resources/carrier/en/976.txt

```diff
@@ -13,9 +13,16 @@
 # limitations under the License.
 
 97680|Unitel
-976830|G-Mobile
-976831|G-Mobile
+97683|G-Mobile
 97685|Mobicom
 97686|Unitel
 97688|Unitel
-97689|Unitel
\ No newline at end of file
+97689|Unitel
```

Comment:
We already had an entry for 97689|Unitel. Please revert this?

Reply:
There's just 1 entry for 97689|Unitel, I don't know why github shows this as a change.

Reply:
If you could see the whole file :)

Reply:
No problem, then we can ignore that :)